prov/tcp/src/tcpx_eq.c

```diff
@@ -61,6 +61,28 @@ static struct fi_ops_eq tcpx_eq_ops = {
 	.strerror = ofi_eq_strerror,
 };
 
+int tcpx_eq_match_ep_event(struct slist_entry *entry, const void *arg)
+{
+	struct tcpx_ep *ep = (struct tcpx_ep *) arg;
+	struct util_event *event;
+	fid_t fid;
+
+	event = container_of(entry, struct util_event, entry);
+
+		fid = (event->err)?
+			((struct fi_eq_err_entry *)event->data)->fid:
+			((struct fi_eq_cm_entry *)event->data)->fid;
```

Comment:
Indentation is off here.  This would be more robust if you could check the event itself to ensure that you can cast data appropriately.